### PR TITLE
[Do not merge,  postpone the release ] Add mysql replica commands

### PR DESCRIPTION
**Pull request by Suna-MS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by troydai** [comment]
@Suna-MS I suggest you close this PR until it is ready to be merged.

**Comment by Suna-MS** [comment]
@troydai, we need to finish the code review before 06/29/2018, then merge it to align with the public review. If I close this PR, will it impact the review process?

File: src/command_modules/azure-cli-rdbms/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.4
```
Comment (QingqingYuan) [inline src/command_modules/azure-cli-rdbms/HISTORY.rst]: New version has been added yesterday. please update this.

Comment (Suna-MS) [reply src/command_modules/azure-cli-rdbms/HISTORY.rst]: Done

**Comment by troydai** [comment]
Review usually takes a few days. Open PR too early will cause you keep making changes to the PR if there is merge conflict.

**Comment by troydai** [comment]
The sprint 40's code complete is 6/27, if you miss that date this change will need to wait till next release.

**Comment by Suna-MS** [comment]
@troydai, we cannot catch up with the 6/27 release. We will wait next release.

**Comment by troydai** [comment]
While you are at it, please resolve the dependencies or rebase the PR against latest dev.

**Comment by Suna-MS** [comment]
Will slip the public preview for weeks for some backend changes, will close this PR and send new then.

**Comment by troydai** [comment]
Thank you for the update.

**Pull request closed without merging**